documentation/source/users/arkane/input.rst

```diff
@@ -113,12 +113,16 @@ Model Chemistry                                  AEC   BC   SOC  Freq Scale Supp
 ``'B3LYP/6-31G(d,p)'``                            v    v         v (0.961)  H, C, O, S
 ``'MRCI+Davidson/aug-cc-pV(T+d)Z'``               v         v               H, C, N, O, S
 ``'wb97x-d/aug-cc-pvtz'``                         v         v               H, C, N, O
+``DLPNO-CCSD(T)/def2-tzvp``                       v    v    v               H, C, N, O, S, F, Cl
+``wb97xd/def2tzvp``                               v    v    v               H, C, N, O, S, F, Cl
 ================================================ ===== ==== ==== ========== ====================
 
 Notes:
 
 - The ``'CBS-QB3-Paraskevas'`` model chemistry is identical to ``'CBS-QB3'`` except for BCs for C/H/O bonds, which are from Paraskevas et al. (DOI: 10.1002/chem.201301381) instead of Petersson et al. (DOI: 10.1063/1.477794). Beware, combining BCs from different sources may lead to unforeseen results.
 - In ``'M08SO/MG3S*'`` the grid size used in the [QChem] electronic structure calculation utilizes 75 radial points and 434 angular points.
+- In ``DLPNO-CCSD(T)/def2-tzvp`` run using Orca 4.2.1 with NormalPNO
```

Comment:
Did you use anything like TightSCF? It can be added here if so. I agree with @alongd it is a great idea to include this here